services/src/kilda-core/kilda-model/src/main/java/org/openkilda/converters/InetSocketAddressConverter.java

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.converters;
+
+import static java.lang.String.format;
+
+import org.neo4j.ogm.typeconversion.CompositeAttributeConverter;
+
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.UnknownHostException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class InetSocketAddressConverter implements CompositeAttributeConverter<InetSocketAddress> {
+
+    private static final String ADDRESS = "address";
+    private static final String PORT = "port";
+
+    @Override
+    public Map<String, ?> toGraphProperties(InetSocketAddress value) {
+        Map<String, Object> address = new HashMap<>();
+        if (value != null) {
+            address.put(ADDRESS, value.getAddress().getHostAddress());
```

Comment:
should we check that `getAddress()` is not null?

Reply:
Added null check